action.yaml

```diff
@@ -13,6 +13,10 @@ inputs:
     description: 'ignore unfixed vulnerabilities'
     required: false
     default: false
+  vuln-type:
+    description: 'list of vulnerability types (os,library)'
```

Comment:
Could you change this to say ` comma-separated list of vulnerability types (os,library)` instead?

Reply:
Done!